spec/lib/secure_headers/headers/policy_management_spec.rb

```diff
@@ -3,6 +3,11 @@
 
 module SecureHeaders
   describe PolicyManagement do
+    before(:each) do
+      reset_config
```

Comment:
We call this in more tests now that it is an exception to reconfigure the default configuration.